src/knock.ts

```diff
@@ -59,6 +73,28 @@ class Knock {
     return this.workflows.trigger(workflowKey, properties);
   }
 
+  signToken(userId: string, options?: SignTokenOptions) {
```

Comment:
I wonder if we should call this `signUserToken` or something more specific? Would be nice to keep all of the terminology aligned